Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/revert.py

```diff
@@ -180,7 +194,7 @@ def create_revert_commit_msg(cls, args, commit_objects, **kwargs):
         else:
             revert_bug_link = revert_issue.link
             for bug in CommitProgram.bug_urls(revert_issue):
-                for regex in cls.RES:
+                for regex in cls.RES_RADAR:
                     if regex.match(bug):
```

Comment:
This seems like it should just be `Tracker.from_string()`, right? Since all of the regexes are centralized in the `Tracker` objects.

Reply:
Good point! I also realized finding both links can be cut down to one iteration of `CommitProgram.bug_urls`.